rdmd.d

```diff
@@ -717,7 +972,7 @@ private string[string] getDependencies(string rootModule, string workDir,
     auto depsGetter =
         // "cd " ~ shellQuote(rootDir) ~ " && "
         [ compiler ] ~ compilerFlags ~
-        ["-v", "-o-", rootModule, "-I" ~ rootDir];
+        ["-v", "-deps", "-o-", rootModule, "-I" ~ rootDir];
```

Comment:
I don't know if I would call it a "regression" :) But yes, it would be a sacrifice in performance for legacy (non `-i`) compilers for the sake of fixing the local import bug.  However, if we do decide to this then we should probably do so in a separate PR.  I'll remove it from this PR.